pkg/kev/converter/kubernetes/transform.go

```diff
@@ -1339,10 +1339,10 @@ func (k *Kubernetes) configEnvs(projectService ProjectService) ([]v1.EnvVar, err
 			})
 		case "pod":
 			// Selects a field of the pod
-			// supported paths: metadata.name, metadata.namespace, metadata.labels, metadata.annotations,
+			// supported paths: metadata.name, metadata.namespace, metadata.labels, metadata.podAnnotations,
```

Comment:
I think this is a leftover?  `metadata.podAnnotations` -> `metadata.annotations` here and in paths below.

Reply:
Yup.